api_views/users.py

```diff
@@ -12,6 +12,15 @@
 def error_message_helper(msg):
     return '{ "status": "fail", "message": "' + msg + '"}'
 
+def error_message_helper1(msg):
+    try:
+        jsonschema.validate(msg, msg)
```

Comment:
Validating a message against itself as schema will always fail and is incorrect usage of jsonschema.validate()